src/class-fee.php

```diff
@@ -325,10 +382,19 @@ function wp() {
 			require_once( ABSPATH . '/wp-admin/includes/post.php' );
 
 			wp_set_post_lock( $post->ID );
-
-			wp_redirect( $this->edit_link( $post->ID ) );
-
-			die;
+			//wp_redirect( $this->edit_link( $post->ID ) );
+			/*
+			 * Fix to avoid redirect loop when clicked on 'Take Over' button
+			 */
+			if ( wp_get_referer() ) {
```

Comment:
Do not use http_referer get the request urls and remove all the request variables.